bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -108,6 +123,8 @@ def update_db_from_file(filename):
     :param filename: path to markdown file.
     :return: SecurityAdvisory instance
     """
+    if HOF_DIRECTORY in filename:
+        return add_hofers(filename, parse_yml_file_base(filename))
```

Comment:
I didn't just because we're not looking for all files in the dir, we've specified two specific file names above. I think the additional check on the filename is superfluous in this case.

Reply:
Okay, sounds good.